tendrl/integrations/ceph/sds_sync/__init__.py

```diff
@@ -0,0 +1,95 @@
+import gevent
+import json
+import os
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message, ExceptionMessage
+from tendrl.commons.utils import cmd_utils
+from tendrl.commons import sds_sync
+from tendrl.node_agent.node_sync import disk_sync
+
+
+class CephIntegrtaionsSyncThread(sds_sync.StateSyncThread):
+    def _run(self):
+        Event(
+            Message(
+                priority="info",
+                publisher=NS.publisher_id,
+                payload={"message": "%s running" % self.__class__.__name__}
+            )
+        )
+        while not self._complete.is_set():
+            # sync of OSD journal details every 30 sec is fine
+            # as this data doesnt change very frequently in cluster
+            gevent.sleep(30)
+            try:
+                # Sync the OSD journal details
+                Event(
+                    Message(
+                        priority="info",
+                        publisher=NS.publisher_id,
+                        payload={
+                            "message":
+                            "ceph_integrations_sync, osd journal details"
+                        }
+                    )
+                )
+                journal_details = {}
+                osd_list = os.listdir("/var/lib/ceph/osd")
```

Comment:
can the user change this path ` /var/lib/ceph/osd` ? If yes, we need to handle that too.

Also why arent you simply using the osd-maps to sync the osd data in tendrl-ceph-integration itself instead of here?